starter-code/apple-pie-starter-code/index.html

```diff
@@ -2,34 +2,59 @@
 <html lang="en">
 <head>
   <title>Apple Pie Recipe</title>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+  <link href="https://fonts.googleapis.com/css?family=Open+Sans:300,600" rel="stylesheet">
+  <link rel="stylesheet" href="./css/main.css">
 </head>
 <body>
-Apple Pies
-This was my grandmother's apple pie recipe. I have never seen another one
-quite like it. It will always be my favorite and has won me several first place
-prizes in local competitions. I hope it becomes one of your favorites as well!
+  
+  <div id="applepie-img">
+    <h1>Apple Pie</h1>  
+  </div>
 
-Ingredients
-1 recipe pastry for a 9 inch double crust pie
-1/2 cup unsalted butter
-3 tablespoons all-purpose flour
-1/4 cup water
-3 tablespoons all-purpose flour
-1/4 cup water
-1/2 cup white sugar
-1/2 cup packed brown sugar
-8 Granny Smith apples - peeled, cored and sliced
+      <div class="container border-bottom">
+        <p>This was my grandmother's apple pie recipe. I have never seen another one
+              quite like it. It will always be my favorite and has won me several first place
+              prizes in local competitions. I hope it becomes one of your favorites as well!</p>
+        <div class="center" >
+        <img src="./images/recipe-info.png">
+            
+        </div>
+      </div>
 
-Directions
-Preheat oven to 425 degrees F (220 degrees C). Melt the butter in a saucepan. Stir in
-flour to form a paste. Add water, white sugar and brown sugar, and bring to a boil. Reduce
-temperature and let simmer.
+      <div class="container border-bottom">
+          <h2 class="padding-bottom">Ingredients</h2>
 
-Place the bottom crust in your pan. Fill with apples, mounded slightly. Cover with a
-lattice work crust. Gently pour the sugar and butter liquid over the crust. Pour slowly so
-that it does not run off.
+          <ul class="padding-bottom">
+            <li>1 recipe pastry for a 9 inch double crust pie1/2 cup unsalted butter</li>
+            <li>3 tablespoons all-purpose flour</li>
+            <li>1/4 cup water</li>
+            <li>3 tablespoons all-purpose flour</li>
+            <li>1/4 cup water</li>
+            <li>1/2 cup white sugar</li>
+            <li>1/2 cup packed brown sugar</li>
+            <li>8 Granny Smith apples - peeled, cored and sliced</li>
+          </ul>
+          <img class="center" src="./images/cooking-info.png">
```

Comment:
Having `./images/cooking-info.png` is the same as having `images/cooking-info.png`